src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -93,11 +93,20 @@ class CredentialType(Enum):  # pylint: disable=too-few-public-methods
     rbac = CLOUD.endpoints.active_directory_graph_resource_id
 
 
+_GLOBAL_CREDS_CACHE = None
```

Comment:
Agreed, but the change is bit bigger than I expected. I opened #3166 to do it at post //build